app/api/event_invoices.py

```diff
@@ -120,13 +131,13 @@ def before_create_object(self, data, view_kwargs):
             data['discount_code_id'] = discount_code.id
 
     view_kwargs = True
-    decorators = (is_admin,)
```

Comment:
I think the list should be accessible only to server admins. However, the organizer should be able to see their own invoices. So the permission would be the organizer but in the before_get we have to ensure if it is direct route ( no event id present ) then user must be admin.